### ISSUE TITLE
o-forms: ensure inline errors are assigned the 'alert' role - ID: DAC_Status_Message_01

### ISSUE THREAD

**Issue by JakeChampion** [Financial-Times/origami#533]
# Issue
This is from page 96 of the [audit report](https://drive.google.com/file/d/17wq6b0CIX41RA4muBFDzUDxE0-qzHBIH/view)

Issue can be found on: <https://www.ft.com/__origami/service/build/v3/demo?component=o-forms@9.0.1&demo=checkboxes&system_code=origami-repo-data&brand=master>

The error message ‘Please select at least one option’ has been placed in a span below the input fields. For sighted users they are able to identify the error message correctly by the colour red but for screen reader users the message is not announced as an alert.
 
# Proposed Solution:
Ensure screen reader users are able to identify when errors have occurred in the checkboxes. In this case, the span which contains the error message should be assigned the role=”alert”. This will enable screen reader users to identify the error message correctly.

**Comment by JakeChampion** [comment]
This issue is also present on <https://www.ft.com/__origami/service/build/v3/demo?component=o-forms@9.0.1&demo=date&system_code=origami-repo-data&brand=master> and found on page 104 of the audit report

**Comment by JakeChampion** [comment]
This issue is also present on <https://www.ft.com/__origami/service/build/v3/demo?component=o-forms@9.0.1&demo=radio-box&system_code=origami-repo-data&brand=master> and found on page 114 of the audit report

**Comment by JakeChampion** [comment]
This issue is also present on <https://www.ft.com/__origami/service/build/v3/demo?component=o-forms@9.0.1&demo=radio-round&system_code=origami-repo-data&brand=master> and found on page 118 of the audit report

ID: DAC_Status_Message_05

**Comment by JakeChampion** [comment]
This issue is also present on <https://www.ft.com/__origami/service/build/v3/demo?component=o-forms@9.0.1&demo=text-input&system_code=origami-repo-data&brand=master> and found on page 133 of the audit report

ID: DAC_Status_Message_06

**Comment by JakeChampion** [comment]
This issue is also present on <https://www.ft.com/__origami/service/build/v3/demo?component=o-forms@9.0.1&demo=file-input&system_code=origami-repo-data&brand=master> and found on page 140 of the audit report

ID: DAC_Status_Message_07

**Comment by JakeChampion** [comment]
This issue is also present on <https://www.ft.com/__origami/service/build/v3/demo?component=o-forms@9.0.1&demo=text-area&system_code=origami-repo-data&brand=master> and found on page 142 of the audit report

ID: DAC_Status_Message_08

**Comment by JakeChampion** [comment]
This issue is also present on <https://www.ft.com/__origami/service/build/v3/demo?component=o-forms@9.0.1&demo=select&system_code=origami-repo-data&brand=master> and found on page 143 of the audit report

ID: DAC_Status_Message_08

**Comment by JakeChampion** [comment]
This issue is also present on <https://www.ft.com/__origami/service/build/v3/demo?component=o-forms@9.0.1&demo=interactive&system_code=origami-repo-data&brand=master> and found on page 159 of the audit report

ID: DAC_Status_Message_09

**Comment by notlee** [comment]
We should init forms JS for all demos as part of this. For example the checkbox error "please select at least one" does not go away after making a selection. If o-forms JS doesn't handle that kind of error out of the box we should avoid writing custom demo code, so users have an accurate representation of the component's capabilities. In this case we could consider updating the error used for the demo
https://www.ft.com/__origami/service/build/v3/demo?component=o-forms@9.0.1&demo=checkboxes&system_code=origami-repo-data&brand=master
